subsys/bindesc/Kconfig.build_time

```
@@ -70,37 +76,45 @@ config BINDESC_BUILD_TIME_STRING
 config BINDESC_BUILD_DATE_TIME_STRING_FORMAT
 	depends on BINDESC_BUILD_DATE_TIME_STRING
 	string "Date-Time format"
-	default "%Y/%m/%d %H:%M:%S"
+	default "%Y-%m-%dT%H:%M:%S%z" if BINDESC_BUILD_TIME_USE_LOCAL_TIME
+	default "%Y-%m-%dT%H:%M:%SZ"
```

```suggestion
	default "%Y-%m-%dT%H:%M:%S%z"
```

As you're proposing a local time option, I'm more than happy to stick with a single format for all (i.e: `%Y-%m-%dT%H:%M:%S%z`)... this also removes accidental use of `Z` where it's not appropriate.

For UTC I'd expect the offset to be `...+0000` in this case.